scripts/build_epub.py

```
@@ -0,0 +1,264 @@
+                            arcname=str(arcname),
+                            compress_type=zipfile.ZIP_DEFLATED
+                        )
+                        file_count += 1
+
+                        # Log every 10th file to avoid spam
+                        if file_count % 10 == 0:
+                            print(f"    Added {file_count} files...")
+
+                print(f"    ✓ Total files added: {file_count + 3}")  # +3 for mimetype, META-INF/*, content.opf
+
+            return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)